FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/History.txt

```diff
@@ -1,3 +1,47 @@
+Changes between V2.2.1 and V2.2.2
+	+ Add CBMC proofs for 4 functions.
```

Comment:
Folks may not know what CBMC is - so better to say something like "added another 4 memory safety proofs using the CBMC model checker".